source/demo-code/recipe-info-rough/recipePage.html

```diff
@@ -8,125 +8,161 @@
     <link rel="stylesheet" href="./recipe-page.css">
     <script defer src="https://use.fontawesome.com/releases/v5.15.4/js/all.js" integrity="sha384-rOA1PnstxnOBLzCLMcre8ybwbTmemjzdNlILg8O7z1lUkLXozs4DHonlDtnE7fpc" crossorigin="anonymous"></script>
     <script src="recipePage.js"></script>
+    <script src="recipeTimer.js"></script>
     <title>Document</title>
 </head>
 <body>
+    <div class="nav">
```

Comment:
What's happening here? This is just a tester html file for the expanded recipe page, but it seems like you made some large changes to it?

Reply:
I have not touched anything other than main and index. So I think we can ignore the demo code( just update to the newer version)

Reply:
I'm not so sure about that, since the version of recipeTimer.js in main.js has more lines than the one in this branch (and seems to be more fleshed out).